public/app/plugins/datasource/mssql/datasource.ts

```diff
@@ -43,14 +44,17 @@ export class MssqlDatasource {
     return quotedValues.join(',');
   }
 
-  interpolateVariablesInQueries(queries: MssqlQueryForInterpolation[]): MssqlQueryForInterpolation[] {
+  interpolateVariablesInQueries(
+    queries: MssqlQueryForInterpolation[],
+    scopedVars: ScopedVars
+  ): MssqlQueryForInterpolation[] {
     let expandedQueries = queries;
     if (queries && queries.length > 0) {
       expandedQueries = queries.map(query => {
         const expandedQuery = {
           ...query,
           datasource: this.name,
-          rawSql: this.templateSrv.replace(query.rawSql, {}, this.interpolateVariable),
+          rawSql: this.templateSrv.replace(query.rawSql, scopedVars, this.interpolateVariable),
         };
```

Comment:
I am going to create a new issue for all sql databases as all of them have following bug. When we want to get to Explore from panel, `interpolateVariablesInQueries` is used and it correctly interpolates variable. You can see correct variables in SQL Editor, but the UI/graphical editor is incorrectly displaying un-interpolated version. When you open Editor and run query, correct results are displayed.
![image](https://user-images.githubusercontent.com/30407135/72990281-2d51c380-3df0-11ea-97ed-5e33c8ce0449.png)
![image](https://user-images.githubusercontent.com/30407135/72990317-3e023980-3df0-11ea-95ab-550e9417fab5.png)